mantidproject/mantid
# ISIS Powder Diffraction system test for Gem

fixes #14357 

New system test for ISIS Powder Diffraction scripts with `GEM`

**To Test**
- Run `ISISPowderDiffractionGemTest.py` & `ISISPowderDiffractionHrpdTest.py` system test locally
- Check all tests pass
- Make sure that all the files are being found within the directory `Testing\Data\SystemTest\GEM`
- Check that `.../analysis/ISISPowderDiffractionTest.py` has been renamed to `...analysis/ISISPowderDiffractionHrpdTest.py`
